torch/csrc/jit/script/parser.cpp

```diff
@@ -256,7 +256,7 @@ struct ParserImpl {
       }
 
       if (kind == TK_FOR) {
-        auto target = parseExp();
+        auto target = parseIdent();
```

Comment:
why targets will be a Ident instead of Expr? things like `a, (b, c) = 1, (2, 3)` will not work. Why do you need to change it in the first place?

Reply:
We don't support that currently so this isn't a regression at least (we just expect everything to be an `Ident` somewhere later in the compilation process. `Expr` includes `in` statements now to keep it from parsing the `in` on a `for-in` loop we need to parse the loop `exprlist` as just identifiers.